shared/constants/metametrics.ts

```diff
@@ -512,6 +512,7 @@ export enum MetaMetricsEventName {
   AppLocked = 'App Locked',
   AppWindowExpanded = 'App Window Expanded',
   BridgeLinkClicked = 'Bridge Link Clicked',
+  BitcoinSupportToggled = 'BTC Experimental Toggled',
```

Comment:
Should we also rename "BTC" here too?

Reply:
Sure

Reply:
[737b269](https://github.com/MetaMask/metamask-extension/pull/25672/commits/737b269ceb50e31b31833ffbefaf65430f182b10)

Reply:
Ah sorry, I was thinking of renaming `BTC` to `Bitcoin` on the right-hand side of this expression (I should have made a suggestion, sorry 🙈)

Reply:
(added 2 suggestions that can be batched together)

Reply:
lol ok